src/components/UserManagement/UserTableData.jsx

```diff
@@ -47,7 +48,7 @@ const UserTableData = React.memo(props => {
       <td className="usermanagement__active--input">
         <ActiveCell
           isActive={props.isActive}
-          canChange={true}
+          canChange={props.canChangeUserStatus}
```

Comment:
```suggestion
          canChange={canChangeUserStatus}
```
Can check permission in the file to decrease prop drilling since the component is already connected and checking for other permissions.